stubs/docutils/docutils/nodes.pyi

```diff
@@ -77,6 +80,25 @@ class Node:
         ascend: bool = False,
     ) -> Node: ...
 
+# Left out
+# - def ensure_str (deprecated)
+# - def unescape (canonical import from docutils.utils)
```

Comment:
Self-explanatory

Reply:
There's a history of leaving out deprecated symbols for third-party stubs, I'd say it's fine.
But *if you want*, you *could* use the `typing_extensions.deprecated` decorator.

Reply:
I think in these two cases, it’s not necessary. One is for Python 2 only and therefore completely unnecessary, the other should just be imported from somewhere else, which is a single line change.